ui/webui/resources/BUILD.gn

```diff
@@ -136,370 +136,373 @@ leo_icons_base_path = "../../../node_modules/@brave/leo/icons"
-  "percent.svg",
-  "picture-in-picture.svg",
-  "play-filled.svg",
-  "plus-add-circle.svg",
-  "plus-add.svg",
-  "previous-outline.svg",
-  "product-bat-outline.svg",
-  "product-bookmarks.svg",
-  "product-brave-leo.svg",
-  "product-brave-news.svg",
-  "product-brave-talk.svg",
-  "product-brave-wallet.svg",
-  "product-ipfs-outline.svg",
-  "product-playlist-add.svg",
-  "product-playlist-bold-add-color.svg",
-  "product-playlist.svg",
-  "product-private-window.svg",
-  "product-sync.svg",
-  "product-translate.svg",
-  "product-vpn.svg",
-  "protocol-handler-off.svg",
-  "protocol-handler.svg",
-  "qr-code.svg",
-  "qwant-color.svg",
-  "radio-checked.svg",
-  "radio-unchecked.svg",
-  "refresh.svg",
-  "remove-circle-filled.svg",
-  "remove-circle-outline.svg",
-  "rewind-15.svg",
-  "rocket.svg",
-  "safe.svg",
-  "search-fuel-tank.svg",
-  "search-zoom-in.svg",
-  "search.svg",
-  "send.svg",
-  "send-filled.svg",
-  "settings.svg",
-  "share-macos.svg",
-  "shield-done.svg",
-  "shield-done-filled.svg",
-  "shield-star.svg",
-  "shuffle-off.svg",
-  "shuffle-on.svg",
-  "shuffle-toggle-on.svg",
-  "sidepanel-open.svg",
-  "sidenav-collapse.svg",
-  "sidenav-expand.svg",
-  "smartphone-desktop-off.svg",
-  "smartphone-desktop.svg",
-  "smartphone-hand.svg",
-  "social-brave-release-favicon-fullheight-color.svg",
-  "social-dribbble.svg",
-  "social-github.svg",
-  "social-reddit.svg",
-  "social-twitch.svg",
-  "social-vimeo.svg",
-  "social-x.svg",
-  "social-youtube.svg",
-  "solana-off.svg",
-  "solana-on.svg",
-  "sort-asc.svg",
-  "sort-desc.svg",
-  "sparkles-off.svg",
-  "sparkles.svg",
-  "stack.svg",
-  "star-outline.svg",
-  "startpage-color.svg",
-  "swap-horizontal.svg",
-  "swap-vertical.svg",
-  "thumb-down.svg",
-  "thumb-up.svg",
-  "trash.svg",
-  "tune.svg",
-  "uphold-color.svg",
-  "usb-off.svg",
-  "usb.svg",
-  "usd-circle.svg",
-  "user-accounts.svg",
-  "user-circle.svg",
-  "user-off.svg",
-  "user.svg",
-  "verification-filled-color.svg",
-  "verification-filled.svg",
-  "video-camera-off.svg",
-  "video-camera.svg",
-  "virtual-reality-off.svg",
-  "virtual-reality.svg",
-  "volume-off.svg",
-  "volume-on.svg",
-  "warning-circle-filled.svg",
-  "warning-circle-outline.svg",
-  "warning-triangle-filled.svg",
-  "warning-triangle-outline.svg",
-  "web3.svg",
-  "web3-blockexplorer.svg",
-  "web3-bridge.svg",
-  "widget-generic.svg",
-  "window-content.svg",
-  "window-tab-close.svg",
-  "window-tab-new.svg",
-  "window.svg",
-  "windows-open.svg",
-  "import-arrow.svg",
-  "metamask-color.svg",
-  "phantom-color.svg",
-  "coinbase-color.svg",
-  "trezor-color.svg",
-  "wallet-ledger.svg",
-  "close-circle.svg",
-  "eth-color.svg",
-  "matic-color.svg",
-  "op-color.svg",
-  "aurora-color.svg",
-  "sol-color.svg",
-  "filecoin-color.svg",
-  "slash.svg",
-  "xmark-color.svg",
-  "yahoo-color.svg",
-  "yandex-color.svg",
-  "window-tabs-vertical-expanded.svg",
-]
-
-if (enable_brave_vpn_panel) {
+leo_icons = []
```

Comment:
it's only useful to do something like this if iOS includes some but not all actual Brave WebUIs